### ISSUE TITLE
Song Manager Volume Control greyed out + in wrong location on GUI

### ISSUE THREAD

**Issue by Malurth** [libertyernie/BrawlManagers#15]
From what I've gathered, there's supposed to be a volume control next to the "add" button, assuming it can find your GCT. For me, it's in the right pane just below the track name controls, and it's disabled, meaning I can't edit it. So "add" does nothing.

![brawl song manager_volume](https://user-images.githubusercontent.com/5057381/41084706-dd45f69a-6a6f-11e8-839a-8dc9f895e1ea.png)

Maybe I'm just misunderstanding the software, but it seems like it's incapable of editing volume at the moment.

**Comment by libertyernie** [comment]
When I click the "add" button, it changes to "remove" and gives me a box with a number:
![image](https://user-images.githubusercontent.com/3732811/41186225-9921b690-6b58-11e8-8808-2d5b65b3e59c.png)
![image](https://user-images.githubusercontent.com/3732811/41186226-a0568238-6b58-11e8-8a6d-9f2c0deceb38.png)
The number ranges from 0 to 127. You have to edit that number, and it will adjust the volume control for you. Unfortunately it doesn't work the other way around.

**Comment by Malurth** [comment]
Yeah, that doesn't happen for me. Here's what it looks like when I click "Add":

![brawl song manager_volume_2](https://user-images.githubusercontent.com/5057381/41508735-3a22c96c-7284-11e8-89aa-bbc3cc18a065.png)

As you can see, all it does is change the text of the button to "Remove," at least as far as the UI is concerned, so it's pretty useless on my end for controlling the volume.

Happens consistently regardless of all the things I've tried.

**Comment by libertyernie** [comment]
The other odd thing is that you don't even get the ???, but you do get the Add/Remove button. Does it say anything in the tooltip if you hover the area to the left of the number?

In the next version I'll get rid of the ??? and make the volume number always visible (but sometimes disabled.) Maybe that will help.

**Comment by Malurth** [comment]
Sadly, there is no tooltip when hovering in that area either.

I'll let you know if the next version allows me to use it then, thanks.

**Comment by libertyernie** [comment]
I just uploaded a new version - could you try it? I don't really expect it to be fixed, but I did get rid of the ??? so maybe you'll see something different.

**Comment by Malurth** [comment]
Doesn't seem to have changed anything on my end, unfortunately. That area is still always empty.

**Comment by libertyernie** [comment]
Version 4.1 has a "Custom Song Volume status" option in the Tools menu. Could you let me know what it says? I'm curious.

**Comment by Malurth** [comment]
Oh, sorry for late response, I completely forgot about this.

And uh...

![customvolume](https://user-images.githubusercontent.com/5057381/43865288-2bbf5636-9b30-11e8-91ec-54b5a10d6241.png)

It just opens a blank window.

(I should note that I'm using a different computer now, but it seems to have the same issues, so no worries there apparently.)
